zingo-testutils/src/lib.rs

```diff
@@ -681,6 +681,69 @@ pub mod scenarios {
         )
     }
 
+    pub async fn funded_orchard_with_3_txs_mobileclient(
+        value: u64,
+    ) -> (RegtestManager, ChildProcessHandler) {
+        let mut scenario_builder = setup::ScenarioBuilder::build_configure_launch(
+            Some(REGSAP_ADDR_FROM_ABANDONART.to_string()),
+            None,
+            Some(20_000),
+        )
+        .await;
+        let faucet = scenario_builder
+            .client_builder
+            .build_new_faucet(0, false)
+            .await;
+        let recipient = scenario_builder
+            .client_builder
+            .build_newseed_client(HOSPITAL_MUSEUM_SEED.to_string(), 0, false)
+            .await;
+        increase_height_and_sync_client(&scenario_builder.regtest_manager, &faucet, 1)
+            .await
+            .unwrap();
+        // received from a faucet
+        faucet
+            .do_send(vec![(
+                &get_base_address!(recipient, "unified"),
+                value,
+                None,
+            )])
+            .await
+            .unwrap();
+        increase_height_and_sync_client(&scenario_builder.regtest_manager, &recipient, 1)
+            .await
+            .unwrap();
+        // send to a faucet
+        recipient
+            .do_send(vec![(
+                &get_base_address!(faucet, "unified"),
+                value.checked_div(10).unwrap(),
+                None,
+            )])
+            .await
+            .unwrap();
+        increase_height_and_sync_client(&scenario_builder.regtest_manager, &recipient, 1)
+            .await
+            .unwrap();
+        // send to self sapling
+        recipient
+            .do_send(vec![(
+                &get_base_address!(recipient, "sapling"),
+                value.checked_div(10).unwrap(),
+                None,
+            )])
+            .await
+            .unwrap();
+        scenario_builder
```

Comment:
I recommend to not use this method *unless* you *want* the block-chain to be ahead of the clients.   I recommend increase_height_and_sync...   instead.

Reply:
syncing would only increase the test time as we dont need the lightclients to be fully synced for mobile, we only care about setting up the blockchain